web/ui/react-app/src/Navbar.tsx

```diff
@@ -25,6 +34,11 @@ const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
       </Link>
       <Collapse isOpen={isOpen} navbar style={{ justifyContent: 'space-between' }}>
         <Nav className="ml-0" navbar>
+          {consoleLink !== '' && (
```

Comment:
A simple test for the existence / non-existence of the extra nav item would be good to have. We don't have a `Navbar.test.tsx` yet, but you can get inspired by e.g. https://github.com/prometheus/prometheus/blob/master/web/ui/react-app/src/pages/flags/Flags.test.tsx